### PR TITLE
PullRequest: mark all new PRs as pending

### LINKED ISSUE #2: Feature request: Mark new PRs (unlabeled) as failure
Something in the lines of

```json
{
  "state": "failure",
  "target_url": "https://ci.riot-labs.de/",
  "description": "\"Ready for CI build\" label not set",
  "context": "Murdock"
}
```

### PR DESCRIPTION
**Pull request by miri64**
Fixes #2.

I did not test it, but this should set every new PR to `pending` immediately with a note, that the PR is awaiting review (I hope that this wording won't solicit newcomers to beg reviewers to set the label ;-)).

### CODE REVIEW

File: murdock.py [hunk 1]
```diff
@@ -145,6 +145,14 @@ def __init__(s, data):
         s.labels = set()
         s.old_head = None
 
+        status = {
+                "state": "pending",
+                "description": "Waiting for review",
+                "context": config.context
+                }
+        s.set_status(s.head, "pending", status)
```
Comment (kaspar030) [inline murdock.py]: This (a little different) should be done in the handler when the "opened" event triggers (line 4xx). But it doesn't. I'll look into it.

Doing it here is a bad idea, as doing things in a constructor which might throw exceptions must be handled, which I don't usually.

Comment (miri64) [inline murdock.py]: Addressed

**Comment by miri64** [comment]
ping?

**Comment by miri64** [comment]
Mh rebasing this PR I see there is already another change in master that does something similar. However this change does not seem to be included in the version that works on the RIOT repository (or it does not work).

**Comment by miri64** [comment]
Maybe because the `state` key wasn't set in the status dict?

**Comment by kaspar030** [comment]
Working in current master.

**Pull request closed without merging**